pkgs/data/misc/colemak-dh/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, stdenvNoCC
+, gzip
+, fetchFromGitHub
+, fetchpatch
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "colemak-dh";
+  version = "0.pre+date=2022-08-07";
+
+  src = fetchFromGitHub {
+    name = "${pname}-${version}";
+
+    owner = "ColemakMods";
+    repo  = "mod-dh";
+    rev = "e846a5bd24d59ed15ba70b3a9d5363a38ca51d09";
+    sha256  = "sha256-RFOpN+tIMfakb7AZN0ock9eq2mytvL0DWedvQV67+ks=";
+    sparseCheckout = "console";
```

Comment:
Why are we doing a sparseCheckout here? That is a bit unusual.

Reply:
The only part I need of the repository is the `console` directory, I believe it simplifies building the package. Is using sparse checkout bad in the scenario?

Reply:
@SuperSandro2000 every other discussion has been solved except this one, should I remove sparseCheckout and instead just do things the traditional way or is this fine?

Reply:
It's fine, in my opinion -- I don't see why it shouldn't be used, looking at the tree.